qiita_pet/handlers/analysis_handlers/base_handlers.py

```diff
@@ -74,9 +74,10 @@ def analysis_description_handler_get_request(analysis_id, user):
     artifacts = {}
     for aid, samples in analysis.samples.items():
         artifact = Artifact(aid)
+        foo = list(set([str(x.id) for x in artifact.prep_templates]))
```

Comment:
Can you use a better name for the "foo" variable? Maybe prep_ids? Also, not sure why we need to convert to list, a set should be fine, no?

Reply:
doh! Thanks for the catch. :)